extern/portable-snippets/COPYING.md

```diff
@@ -0,0 +1,124 @@
+Each source file contains a preamble explaining the license situation
```

Comment:
I don't see any requirement to have this license file in the sources... License itself is already mentioned in the header file

Reply:
Removed.